src/metric/client.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, region):
         session = get_session()
         if container_creds is not None:
             session._credentials = container_creds
-            self.client = boto3.Session(botocore_session=session).client('cloudwatch', region)
+            self.client = boto3.Session(botocore_session=session).client(
+                'cloudwatch', region, config=config.Config(proxies_config={'proxy_ca_bundle': utils.GG_ROOT_CA_PATH}))

```

Comment:
How do we ensure we're using a compatible boto3 version, since they only just fixed it?

Reply:
For 3.0.x , we do a pip install in comp recipe which pulls in latest boto3. For cases where customer already has boto3 installed with a previous version, we will be specifying the requirements in docs about the minimum version required

Reply:
we can bump this to a 3.1.0 too - as this is a new feature